packages/fs-detectors/src/workspaces/get-workspace-package-paths.ts

```diff
@@ -127,3 +138,23 @@ async function getPnpmWorkspacePackagePaths({
 
   return getPackagePaths(packages, fs);
 }
+
+async function getRushWorkspacePackagePaths({
+  fs,
+}: GetPackagePathOptions): Promise<string[]> {
+  const rushWorkspaceAsBuffer = await fs.readFile('rush.json');
+
+  const { projects = [] } = JSON.parse(
+    rushWorkspaceAsBuffer.toString()
+  ) as RushWorkspaces;
+
+  if (Array.isArray(projects)) {
+    const packages = projects
+      .filter(proj => proj.projectFolder)
+      .map(project => project.projectFolder);
```

Comment:
This is where the [previous PR](https://github.com/vercel/vercel/pull/8565/files#diff-b318a167de8653a693a7d6029d515e14dadae456c9e5834debb60c40e9077fa7R152) had a preceding `/` added to the project directory, which caused the later `glob` to return results with the windows prefix (like `C:`), which was failing the test assertions.

Reply:
@chloetedder Any reason this is using `proj` for the filter but `project` for the map?